R/pkg/R/sparkR.R

```diff
@@ -365,6 +365,10 @@ sparkR.session <- function(
     }
     overrideEnvs(sparkConfigMap, paramMap)
   }
+  if (nzchar(master)) {
+    assign("spark.master", master, envir = sparkConfigMap)
```

Comment:
ok I see. So then starting programmatically has been broken from early on then.
@shivaram I think we need to port this to 2.0.1?